soccer/Processor.cpp

```diff
@@ -386,6 +386,16 @@ void Processor::run() {
             delete packet;
         }
 
+        // Log referee data
+        vector<NewRefereePacket*> refereePackets;
+        _refereeModule.get()->getPackets(refereePackets);
+        fprintf(stderr, "Ref packets received this loop:%lu\n",
+                refereePackets.size());
+        for (NewRefereePacket* packet : refereePackets) {
+            SSL_Referee* log = _state.logFrame->add_raw_refbox();
+            log->CopyFrom(packet->wrapper);
+        }
```

Comment:
Btw, it looks like vision packets do get deleted after they're used: https://github.com/robojackets/robocup-software/blob/ryan/bsv-export/soccer/Processor.cpp#L386.

Smart pointers would be preferable there too, but it's not a big deal.